conans/client/conan_api.py

```diff
@@ -66,6 +66,7 @@ def wrapper(*args, **kwargs):
         try:
             log_command(f.__name__, kwargs)
             the_self._init_manager()
+            curdir = os.getcwd()
```

Comment:
Should be first line, before ``_init_manager()``. If init_manager fails, the finally will also raise because curdir not defined.